server/tonton_sommelierBDD.sql

```diff
@@ -1,9 +0,0 @@
-CREATE DATABASE tonton_sommelier;
-
-USE tonton_sommelier;
-
-CREATE TABLE `box`(
-	`id` INT NOT NULL AUTO_INCREMENT,
-	`name` VARCHAR (100) NOT NULL,
-	PRIMARY KEY (`id`)
-);
```

Comment:
why removing the table?

Reply:
we keep "archive.js" with all commands but this one is remove when we don't have to update the database